lib/apivore/rspec_matchers.rb

```diff
@@ -35,13 +35,25 @@ module RspecMatchers
       end
     end
 
-    matcher :be_consistent_with_swagger_definitions do |master_swagger|
+    matcher :be_consistent_with_swagger_definitions do |master_swagger, current_service|
 
       attr_reader :actual, :expected
 
+      def cleaned_definitions(definitions, current_service)
+        definitions.transform_values do |definition_fields|
+          # We ignore definitions that are owned exclusively by the current_service
+          if [current_service] == definition_fields['x-services']
+            nil
+          else
+            # 'x-services' is added by api.westfield.io - services shouldn't need to define it
+            definition_fields.tap{|d|d.delete 'x-services'}
```

Comment:
If we remove x-services we will not be able to report which services we conflicted with. Is that correct?